src/main/steps/sequence/have-you-applied-for-help-with-fees/content.ts

```diff
@@ -7,11 +7,11 @@ import { commonContent } from '../../common/common.content';
 export const generateContent: TranslationFn = ({ isDivorce }) => {
   const en = {
     title: `Have you already applied for help with your ${isDivorce ? 'divorce' : ''} fee?`,
-    yes: 'Yes',
+    yes: commonContent.en.yes,
     enterRefNo: 'Enter your Help With Fees reference number:',
     refReceivedWhenApplied: 'You received this when you applied for help with your fees.',
     refExample: 'For example, HWF-A1B-23C',
-    no: 'No',
+    no: commonContent.en.no,
```

Comment:
you dont need these as common is also passed in with the template render and this will exist. same with the yes property and same in welsh.